docker/proxy-init.Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG BASEIMAGE=registry.k8s.io/build-image/distroless-iptables:v0.2.3
+ARG BASEIMAGE=registry.k8s.io/build-image/distroless-iptables:v0.2.4
```

Comment:
wish we could automate this. should we just remove the arg so dependabot can bump, wdyt?

Reply:
sgtm! pushed https://github.com/Azure/azure-workload-identity/pull/946/commits/729b15877ec5a4591fe02d46206fb893d54ad15d to remove the `ARG`.